Sources/Kitura/CodableRouter.swift

```diff
@@ -434,6 +434,8 @@ public struct CodableHelpers {
      *            if valid, or HTTPStatusCode.unknown otherwise
      */
     public static func httpStatusCode(from error: RequestError) -> HTTPStatusCode {
+        // ORM status code 7XX mapped to internalServerError 500
+        if error.httpCode >= 700 && error.httpCode < 800 { return HTTPStatusCode(rawValue: 500) ?? HTTPStatusCode.unknown }
```

Comment:
Should just `return .internalServerError`

Reply:
could do

Reply:
could no use `.internalServerError`... had to use `HTTPStatudCode.internalServerError`... it should be the same thing